metron-platform/metron-parsers/README.md

```diff
@@ -216,6 +216,23 @@ whenever `field2` exists and whose corresponding equal to 'foo':
 }
 ```
 
+* `SELECT`: This transformation filters the fields in the message to include only the configured output fields, and drops any not explicitly included. 
+
+For example: 
+```
+{
+...
+    "fieldTransformations" : [
+          {
+            "output" : ["field1", "field2" ] 
+          , "transformation" : "SELECT"
+          }
+                      ]
+}
+```
+
+when applied to a message containing keys field1, field2 and field3, will only output the first two.
+
```

Comment:
How does this relate to REMOVE?  If I have more than one transformation should this be last or does it not matter?

Reply:
This is the opposite of remove.

Reply:
And order doesn't really matter, you might have this first, or last, or sandwiched between two stellar blocks. That said, I would expect the most common use case to be have this one at the end of a set.

Reply:
If a user has a stellar transform (existing ) that takes field X as input, but doesn't want X as output, and puts the SELECT in the wrong order, could they break their existing transform because X now missing?

Reply:
Yes, that would break. I'd say that was flat out user error, and not something to guard against, or indeed something it's possible to guard against. I guess the key thing here is that it requires maintaining order in field transformations.

Reply:
It should be documented then in the readme

Reply:
We should document that the system fields are protected as well

Reply:
agreed, and added.